pkg/cli/cli.go

```diff
@@ -390,50 +463,10 @@ func (c cli) buildRootCmd() *cobra.Command {
 // defaultCommand returns the root command without its subcommands.
 func (c cli) defaultCommand() *cobra.Command {
```

Comment:
Being now a direct `return`, only a couple lines long, and just being called from one place, I wonder if it makes sense to have it as a separate function or we could just create it in place. Also, `defaultCommand` does not make sense any more as the function name as previously it was providing default descriptors but now we can customize them and `c.cdm` already has the "final" descriptors.